src/content/add-to-app/ios/project-setup.md

```diff
@@ -104,46 +99,55 @@ directory. Changes made in your module's `.ios/`
 directory don't appear in your existing iOS project
 using the module, and might be overwritten by Flutter.
 
-Do not source control the `.ios/` directory since it's autogenerated.
+Exclude the `.ios/` directory from source control as it's autogenerated.
 Before building the module on a new machine, run `flutter pub get`
```

Comment:
This is really important information, but currently it's buried as the second sentence in the second paragraph of a note. Should we make this it's own separate note?